[CI] ChangeDetectorTests testMultipleChanges failing

**Build Scans:**
- [elasticsearch-periodic #6061 / encryption-at-rest](https://gradle-enterprise.elastic.co/s/7l6ptmjlyps4o)
- [elasticsearch-periodic-platform-support #5960 / ubuntu-1804_platform-support-unix](https://gradle-enterprise.elastic.co/s/7tiilwme7vdck)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:ml:test" --tests "org.elasticsearch.xpack.ml.aggs.changepoint.ChangeDetectorTests.testMultipleChanges" -Dtests.seed=E322DF8C5BE7858A -Dtests.locale=uz-UZ -Dtests.timezone=Australia/West -Druntime.java=23
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.ml.aggs.changepoint.ChangeDetectorTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testMultipleChanges),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a value greater than <90>
     but: <90> was equal to <90>
```

**Issue Reasons:**
- [8.x] 2 failures in test testMultipleChanges (0.2% fail rate in 973 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/aggs/changepoint/ChangeDetectorTests.java
+++ b/x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/aggs/changepoint/ChangeDetectorTests.java
@@ -192,7 +192,7 @@ public class ChangeDetectorTests extends AggregatorTestCase {
             ChangeType type = new ChangeDetector(bucketValues).detect(0.05);
             tp += type instanceof ChangeType.TrendChange ? 1 : 0;
         }
-        assertThat(tp, greaterThan(90));
+        assertThat(tp, greaterThan(80));
     }
 
     public void testProblemDistributionChange() {

